poetry/orb.yaml

```diff
@@ -105,7 +111,7 @@ jobs:
             fi
           working_directory: <<parameters.cwd>>
       - run:
-          command: poetry build
+          command: if [[ <<parameters.skip_build>> == true ]]; then echo "Skipping build"; else poetry build; fi
```

Comment:
There's actually a CircleCI `when` builtin for handling this case: [docs](https://circleci.com/docs/2.0/configuration-reference/#the-when-step-requires-version-21). That'll let Circle avoid needing to even parse that step out at runtime.

Reply:
@TheKevJames any advice on bumping the orb version Ive never done this

Reply:
You mean to release your changes? Once this is merged, you'll pull `master` and tag it with `poetry-minor`. [See the README](https://github.com/talkiq/circleci-orbs#development)

Reply:
Oh -- I see you bumped the `version` specifier within the orb.yaml. That's the _CircleCI Orb yaml syntax_ version number: 2.1 is latest and should have all the features you need.